app/models/user.rb

```diff
@@ -1,4 +1,16 @@
 
+class User < ApplicationRecord
+  validates_presence_of :name, :address,
+                        :city, :state,
```

Comment:
we should add validates presence of role too I think, I might add in a later pull request.